### PR TITLE
Allow resizing columns in CodeMetrics

### LINKED ISSUE #4102: Code metrics resize window
I get less space for the code in the left pane than for the brief metrics in the right pane. I'd like to resize them.  2.2.0.3398-pre

### PR DESCRIPTION
**Pull request by IvenBach**
Close #4102. As described in the title.

- R# flagged several items that aren't being used, `BitmapImage`s and `converters`. Are these yet to be implemented?
- In navigating Rubberduck>Tools>CodeMetrics I noticed it didn't have a hotkey assigned. Any objection to assigning M as the hotkey? Export active project doesn't have a hotkey assigned either.

### CODE REVIEW

**Comment by retailcoder** [comment]
IIRC Ctrl+M is the default hotkey for "indent module" (carried from legacy Smart Indenter hotkeys). Adding a hotkey for code metrics would be awesome - just make sure to pick one that's not already in use :smile:

**Comment by IvenBach** [comment]
untitled

@retailcoder Incorrect word usage, my fault. Thought I was explicit enough with Rubberduck>Tools>CodeMetrics to refer to the above image. That was what I'd meant.

File: Rubberduck.Core/UI/CodeMetrics/CodeMetricsControl.xaml [hunk 1]
```diff
@@ -233,20 +234,20 @@
         </ResourceDictionary>
     </UserControl.Resources>
 
-    <Grid UseLayoutRounding="True">
+    <Grid UseLayoutRounding="True"
+          Background="White">
```
[the remark this answers is not in the document]
Comment (IvenBach) [reply Rubberduck.Core/UI/CodeMetrics/CodeMetricsControl.xaml]: I'd pulled it out from a nested element. Turns out it wasn't needed. F5 tested and saw no difference.

File: Rubberduck.Parsing/VBA/RubberduckParserState.cs [hunk 2]
```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Runtime.InteropServices;
 using System.Threading;
+using Antlr4.Runtime;
```
Comment (Vogel612) [inline Rubberduck.Parsing/VBA/RubberduckParserState.cs]: huh? Where does that come from?

File: Rubberduck.Core/UI/CodeMetrics/CodeMetricsControl.xaml [hunk 3]
```diff
@@ -282,8 +286,8 @@
             </ListView.ItemTemplate>
         </ListView>
 
-        <controls:EmptyUIRefresh Grid.ColumnSpan="2" Grid.RowSpan="2" Visibility="{Binding EmptyUIRefreshMessageVisibility, Mode=OneWay, Converter={StaticResource BoolToVisibility}}" />
-        <controls:BusyIndicator Grid.ColumnSpan="2" Grid.RowSpan="2" Width="120" Height="120" Visibility="{Binding IsBusy, Mode=OneWay, Converter={StaticResource BoolToVisibility}}" />
+        <controls:EmptyUIRefresh Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2" Grid.RowSpan="2" Visibility="{Binding EmptyUIRefreshMessageVisibility, Mode=OneWay, Converter={StaticResource BoolToVisibility}}" />
+        <controls:BusyIndicator Grid.Row="0" Grid.Column="0" Grid.ColumnSpan="2" Grid.RowSpan="2" Width="120" Height="120" Visibility="{Binding IsBusy, Mode=OneWay, Converter={StaticResource BoolToVisibility}}" />
```
Comment (Vogel612) [inline Rubberduck.Core/UI/CodeMetrics/CodeMetricsControl.xaml]: The Refresh indicator should cover the whole grid. Could we get a ColumnSpan="3"?

File: Rubberduck.Parsing/VBA/RubberduckParserState.cs (hunk 2 above)
Comment (bclothier) [reply Rubberduck.Parsing/VBA/RubberduckParserState.cs]: Not for this PR but a prior merge apparently lost that change; the `next` now can't build without that `using`.

**Pull request merged**